backend/api/Database/Models/ReportEntry.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+# nullable disable
+namespace Api.Models
+{
+
+    public class ReportEntry
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Required]
+        public string Id { get; set; }
+        [Required]
+        public virtual Report Report { get; set; }
+        public string TagId { get; set; }
+        [Required]
+        public ReportEntryStatus ReportEntryStatus { get; set; }
+        [Required]
+        public InspectionType InspectionType { get; set; }
+        [Required]
+        public DateTimeOffset Time { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string FileLocation { get; set; }
+    }
+
+    public enum ReportEntryStatus
+    {
+        Completed, Failed
+    }
+
+    public enum InspectionType
+    {
+        Image, ThermalImage
+    }
```

Comment:
See previous comment on string enums. Open for discussion.

Reply:
Solved it by adding a json serializer to the controller:
`builder.Services.AddControllers().AddJsonOptions(options => options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));`